recipe/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
 
 build:
   number: 0
-  # the upstream package does not support windows or python2.7
-  skip: true  # [win or py<36]
+  # embree requires visual studio 13+
```

Comment:
What configurations does this actually affect? Does anything use Visual Studio <13?

Reply:
I have this selector in the `embree3` feedstock. It was required at one point because conda-forge had a build configuration using a visual studio (I think 2012) that was unsupported by embree.

I just tested removing it and `conda smithy rerender` doesn't generate any new build configurations so this is not needed.